README.md

```diff
@@ -74,6 +74,7 @@ Example config to run Aye Spy:
         "label": "homepage",
         "cropToSelector": ".flickity-slider", // crop the screenshot to a specific selector
         "removeElements": ["#ad-header"], // remove elements that are not static on refresh such as adverts
+        "zeroOpacityElements": [".is-delayedImage"] // set zero opacity on elements, content will be still there but not visible
```

Comment:
// hide elements that are not static on refresh such as adverts

Reply:
Sam and think it would be good to call `zeroOpacityElements`
`hideElements`

I think it's a bit more descriptive

Reply:
updated in latest version